grunt/requirejs.js

```diff
@@ -13,7 +13,7 @@ module.exports = function( grunt, options ) {
                 mainConfigFile: '<%= srcPath %>components/app/main.js',
                 dir: '<%= distPath %>js',
 
-                fileExclusionRegExp: /\.(njs|scss|gitkeep)/,
+                fileExclusionRegExp: /\.(njs|scss|gitkeep|DS_Store)/,
```

Comment:
Do we really need this?

Reply:
Plugin grunt-contrib-requirejs threw an exception because it took this file into account. Since i can't reproduce this, i undo this change.